From f5c771d07a863bb2d70bdfb39ffa938176da8756 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 1 Sep 2016 15:18:15 -0700
Subject: [PATCH] RELENG-715 ironically the "ENG" jira project is not an
 engineering project Reviewed by: Orlando Vazquez <orlando@joyent.com>
 Approved by: Orlando Vazquez <orlando@joyent.com>

---
 tools/jira/listengprojects.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/jira/listengprojects.sh b/tools/jira/listengprojects.sh
index b54aa8a..2ab1e3d 100755
--- a/tools/jira/listengprojects.sh
+++ b/tools/jira/listengprojects.sh
@@ -46,7 +46,6 @@ DATASET
 DCAPI
 DOC
 DOCKER
-ENG
 FWAPI
 HEAD
 HVM
-- 
2.21.0

